File: src/main/java/bank/Account.java
package bank;

import java.util.ArrayList;
import java.util.List;

public class Account {
    private final List<Movement> movements;
    private final AccountId accountId;

    public Account(AccountId accountId) {
        this.accountId = accountId;
        movements = new ArrayList<>();
    }

    public void add(Movement movement) {
        movements.add(movement);
    }

    public Float getLastMovement() {
        final Movement movement = movements.get(movements.size() - 1);
        return movement.getAmount();
    }

    public List<Movement> getAllMovements() {
        return movements;
    }

    public AccountId getAccountId() {
        return accountId;
    }
}

File: Prevoir/Account.cs
using System.Collections.Generic;

namespace Prevoir
{
    public class Account
    {
        public List<Movement> Movements { get; }
        public AccountId Id { get; }

        public Account(AccountId accountId)
        {
            Id = accountId;
            Movements = new List<Movement>();
        }

        public void Add(Movement movement)
        {
            Movements.Add(movement);
        }

        public float getLastMovement()
        {
            Movement movement = Movements[Movements.Count - 1];
            return movement.Amount;
        }

        public List<Movement> GetAllMovements()
        {
            return Movements;
        }
    }
}